### PR TITLE
Update Google Slides component editor to match UI specs [stage-4]

### PR DESCRIPTION
**Pull request by andrecardoso**
Screen Shot 2019-06-12 at 6 16 36 PM
Screen Shot 2019-06-12 at 6 17 04 PM
Screen Shot 2019-06-12 at 6 17 24 PM

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Apply styles to match UI spec](https://trello.com/c/4cz8yEMA/112-2-apply-styles-to-match-ui-spec)

File: web/partials/template-editor/components/component-slides.html [hunk 1]
```diff
@@ -1,18 +1,23 @@
 <div class="row attribute-row-first">
-  <label class="control-label">Google Slides url</label>
-  <div ng-class="{ 'has-error': validationError }">
-    <input type="text" class="form-control" ng-model="src" placeholder="Enter URL" ng-keypress="handleKeyPressSrc($event)" ng-model-options="{ debounce: 1000 }" ng-change="saveSrc()" >
-    <p class="help-block" ng-switch on="validationError">
+  <div class="form-group has-feedback" ng-class="{ 'has-error': validationResult && validationResult !== 'VALID', 'has-success': validationResult === 'VALID' }">
+    <label class="control-label" for="urlInput">Google Slides url</label>
```
Comment (olegrise) [inline web/partials/template-editor/components/component-slides.html]: @andrecardoso could you please change `url` to `URL` as we discussed?

File: web/partials/template-editor/components/component-slides.html [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+    </p>
   </div>
 </div>
 <div class="attribute-row-last">
   <label class="control-label">Show each slide for</label>
-  <div class="input-group">
+  <div class="input-group input-duration">
     <input type="number" class="form-control" ng-model="duration" placeholder="Enter duration" ng-change="saveDiration()">
-    <span class="input-group-addon">seconds</span>
+    <span class="input-group-addon">Seconds</span>
```
Comment (olegrise) [inline web/partials/template-editor/components/component-slides.html]: @andrecardoso as we discussed, I think it should be `seconds`.

**Comment by andrecardoso** [comment]
@olegrise please re-review. Thanks

**Pull request merged**